src/modules/battery.cpp

```diff
@@ -68,9 +83,11 @@ void waybar::modules::Battery::worker() {
     refreshBatteries();
     dp.emit();
   };
+#endif
 }
 
 void waybar::modules::Battery::refreshBatteries() {
+#if (__Linux__)
```

Comment:
Witthout `defined` may trigger `-Wundef` on FreeBSD and should be lowercase:
```suggestion
#if defined(__linux__)
```